--- scummvm-1.1.0/scummvm-tools-1.1.0/configure~	2010-04-02 19:55:43.000000000 +0200
+++ scummvm-1.1.0/scummvm-tools-1.1.0/configure	2010-04-20 12:46:30.628249474 +0200
@@ -180,7 +180,7 @@
 # TODO: small bit of code to test wxWidgets usability
 find_wxconfig() {
 	echo_n "Looking for wx-config... "
-	wxconfigs="$_wxconfig:wxgtk2-2.8-config"
+	wxconfigs="$_wxconfig:wx-gtk3-unicode-config"
 	_wxconfig=
 
 	IFS="${IFS=   }"; ac_save_ifs="$IFS"; IFS="$SEPARATOR"
