diff -ruN scummvm-0.6.0./scumm/actor.cpp scummvm-0.6.0/scumm/actor.cpp
--- scummvm-0.6.0./scumm/actor.cpp	2004-02-25 05:37:28.000000000 +0100
+++ scummvm-0.6.0/scumm/actor.cpp	2004-03-16 16:25:09.866004528 +0100
@@ -789,14 +789,14 @@
 		return _V1_talkingActor;
 	else
 		return VAR(VAR_TALK_ACTOR);
-};
+}
 
 void ScummEngine::talkingActor(int value) {
 	if (_gameId == GID_MANIAC && _version == 1)
 		_V1_talkingActor = value;
 	else
 		VAR(VAR_TALK_ACTOR) = value;
-};
+}
 
 void ScummEngine::showActors() {
 	int i;
diff -ruN scummvm-0.6.0./scumm/debugger.cpp scummvm-0.6.0/scumm/debugger.cpp
--- scummvm-0.6.0./scumm/debugger.cpp	2004-02-13 11:26:40.000000000 +0100
+++ scummvm-0.6.0/scumm/debugger.cpp	2004-03-16 16:25:30.984793984 +0100
@@ -57,7 +57,7 @@
 	va_end(va);
 
 	debug(buf);
-};
+}
 	
 ScummDebugger::ScummDebugger(ScummEngine *s)
 	: Common::Debugger<ScummDebugger>() {
@@ -592,7 +592,7 @@
 
 	DebugPrintf("Unknown flag. Type 'Debug ?' for syntax\n");
 	return true;
-};
+}
 
 bool ScummDebugger::Cmd_DebugLevel(int argc, const char **argv) {
 	if (argc == 1) {
diff -ruN scummvm-0.6.0./base/main.cpp scummvm-0.6.0/base/main.cpp
--- scummvm-0.6.0./base/main.cpp	2004-03-14 14:45:12.000000000 +0100
+++ scummvm-0.6.0/base/main.cpp	2004-03-16 16:28:53.712974616 +0100
@@ -262,7 +262,7 @@
 
 	// Free up memory
 	delete engine;
-};
+}
 
 #ifndef _WIN32_WCE
 int main(int argc, char *argv[]) {
