--- scummvm-0.12.0/scummvm-tools-0.12.0/Makefile~	2008-08-26 01:23:03.000000000 +0200
+++ scummvm-0.12.0/scummvm-tools-0.12.0/Makefile	2008-12-26 11:38:58.571411354 +0100
@@ -190,14 +190,14 @@
 	$(CXX) $(LDFLAGS) -o $@ $+
 
 tools_gui$(EXEEXT): tools_gui.o
-	$(CXX) $(LDFLAGS) -o $@ $+ `wx-config --libs`
+	$(CXX) $(LDFLAGS) -o $@ $+ `wx-gtk2-unicode-config --libs`
 
 
 clean:
 	rm -f *.o utils/*.o $(TARGETS)
 
 tools_gui.o: tools_gui.cpp tools_gui.h
-	$(CXX) $(CXXFLAGS) $(CPPFLAGS) `wx-config --cxxflags` -c tools_gui.cpp -o tools_gui.o
+	$(CXX) $(CXXFLAGS) $(CPPFLAGS) `wx-gtk2-unicode-config --cxxflags` -c tools_gui.cpp -o tools_gui.o
 
 
 ######################################################################
