diff -ruN scummvm-/tools/extract.c scummvm/tools/extract.c
--- scummvm-/tools/extract.c	Mon Sep  9 07:46:02 2002
+++ scummvm/tools/extract.c	Thu Sep 19 16:20:03 2002
@@ -221,7 +221,7 @@
 		if (encparms.silent == 1)
 			strcat(fbuf_temp," --silent");
 		sprintf(fbuf,
-		        "lame -t -q %i %s -V %i -B %i --resample 22.05 -m m -r -s %d %s %s",
+		        "lame -t -q %i %s -V %i -B %i --resample 22.05 -m m -r --athtype 3 --lowpass 21500 -s %d %s %s",
 		        encparms.algqual, fbuf_temp, encparms.vbrqual,
 		        encparms.maxBitr, real_samplerate, rawname, mp3name);
 		system(fbuf);
