diff -ruN scummvm-0.3.0-/Makefile scummvm-0.3.0/Makefile
--- scummvm-0.3.0-/Makefile	Sun Dec  1 21:29:19 2002
+++ scummvm-0.3.0/Makefile	Sun Dec  8 16:50:46 2002
@@ -40,8 +40,8 @@
 LIBS    += -lmad
 
 # Uncomment this to activate the Ogg Vorbis lib for compressed sound files
-# DEFINES += -DUSE_VORBIS
-# LIBS    += -lvorbisfile -lvorbis
+DEFINES += -DUSE_VORBIS
+LIBS    += -lvorbisfile -lvorbis
 
 # Uncomment this to activate the ALSA lib for midi
 # DEFINES += -DUSE_ALSA
