diff -ruN scummvm-0.8.0./simon/simon.cpp scummvm-0.8.0/simon/simon.cpp
--- scummvm-0.8.0./simon/simon.cpp	2005-10-23 15:45:10.000000000 +0200
+++ scummvm-0.8.0/simon/simon.cpp	2005-10-31 20:11:43.200089576 +0100
@@ -345,11 +345,6 @@
 				sprintf(md5str + j*2, "%02x", (int)md5sum[j]);
 			}
 
-			for (j = 0; j < 3; j++) {
-				if (!strcmp(md5str, bad_versions[j]))
-					error("Cracked versions aren't supported");
-			}
-
 			printf("%s  %s\n", md5str, buf);
 			const MD5Table *elem;
 			elem = (const MD5Table *)bsearch(md5str, md5table, ARRAYSIZE(md5table)-1, sizeof(MD5Table), compareMD5Table);
