--- scummvm-2.5.1/configure.orig	2022-09-11 22:50:44.875895176 +0200
+++ scummvm-2.5.1/configure	2022-09-11 23:12:09.358791198 +0200
@@ -3840,7 +3840,7 @@
 	append_var INCLUDES "$SDL_CFLAGS"
 	append_var LIBS "$SDL_LIBS"
 	case $_sdlversion in
-		2.0.*)
+		2.*)
 			add_line_to_config_mk "USE_SDL2 = 1"
 			_sdlMajorVersionNumber=2
 			;;
@@ -5368,7 +5368,7 @@
 					_opengl_mode=gl
 					;;
 
-				2.0.*)
+				2.*)
 					# SDL2 supports both OpenGL + OpenGL ES contexts.
 					# However, Mac OS X only allows OpenGL context creation at
 					# this time, thus we limit us to OpenGL on that platform.
